Documentation/Books/Manual/ReleaseNotes/KnownIssues32.md

```diff
@@ -97,5 +97,5 @@ The following known issues will be resolved in future releases:
 
 ### OpenSSL 1.1
 
- * ArangoDB has been tested with OpenSSL 1.0 only and won't build against 1.1 when compiling on your own. See [here](../../cookbook/Compiling/OpenSSL.html)
+ * ArangoDB has been tested with OpenSSL 1.0 only and won't build against 1.1 when compiling on your own. See [here](../../Cookbook/Compiling/OpenSSL.html)
```

Comment:
you should **NOT** use `here` as link target. This is very bad for screen readers. Links should cary a descriptive text.

Reply:
I totally agree. However, we have this all over the place, also on the website if I'm not mistaken. I'll create an internal ticket to look into this.